native/src/main/java/io/ballerina/stdlib/http/transport/contractimpl/sender/HttpClientChannelInitializer.java

```diff
@@ -219,7 +219,8 @@ private void configureSslForHttp2(SocketChannel ch, ChannelPipeline clientPipeli
                                 sslConfig.getCacheSize()));
             }
         }
-        clientPipeline.addLast(new Http2PipelineConfiguratorForClient(targetHandler, connectionAvailabilityFuture));
+        clientPipeline.addLast(
+                new Http2PipelineConfiguratorAfterALPNNegotationForClient(targetHandler, connectionAvailabilityFuture));
```

Comment:
This name seems to be too long and this handler can set the pipeline based on the ALPN negotiation. So how about calling this as `ALPNClientHandler`?

Reply:
Updated